engine/src/main/java/org/terasology/rendering/opengl/BaseFBOsManager.java

```diff
@@ -32,6 +32,8 @@
 
     FBO get(ResourceUrn fboName);
```

Comment:
The reason I suggest this is the code-side implementation.
fboManager.getFbo(SCENE_FINAL) is more clear than fboManager.get(SCENE_FINAL), since we also have a fboManager.getFboConfig(SCENE_FINAL) method now, so what the fboManager is returning isn't super obvious.
But sure, I'll leave it as it is.